### ISSUE TITLE
logging events are triggering negative CWD flux?

### ISSUE THREAD

**Issue by rgknox** [NGEET/fates#527]
I'm testing a logging enabled run. This uses the default parameter set, which triggers logging on the 30th day of every year.

This uses master, as of tag sci.1.26.2_api.7.2.0

The logging event triggers at the correct time, and the event does indeed generate the disturbance (ie it generates more than fire/treefall).

However, I'm noticing lots of these diagnostics in the log:

```
negative CWD in flux  -12.364351702890730        2259.4749780927568        7.6772113082213501E-003
 negative CWD in flux  -28.219373800318380        2259.4749780927568        0.0000000000000000     
 negative CWD in flux  -26.110139480480388        3201.1677324999446        6.3600981656457764E-003
 negative CWD in flux  -45.111401249776783        3201.1677324999446        0.0000000000000000     


```

@huangmy

**Comment by rgknox** [comment]
this error is suspiciously close to the fixes in #526, hopefully that PR addresses this issue. Testing now.

**Comment by rgknox** [comment]
this was resolved in #526
